nixos/modules/services/networking/peroxide.nix

```diff
@@ -0,0 +1,180 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.peroxide;
+  settingsFormat = pkgs.formats.yaml { };
+in
+{
+  options.services.peroxide = {
+    enable = mkEnableOption "peroxide service";
+
+    stateDir = mkOption {
+      type = types.str;
+      default = "/var/lib/peroxide";
+      description = ''
+        The state directory, where the cookie-jar, certs, credential store, and
+        log files live.
+      '';
+    };
```

Comment:
If we are using StateDir in systemd this must be under /var/lib. I am also not sure if this needs to be configurable at all.

Reply:
Searching for `stateDir` on search.nixos.org/options returns only a single page worth of results, so making it configurable does not appear to be the norm.  I'm happy to move it to a `let` binding.